src/engraving/dom/joinMeasure.cpp

```diff
@@ -55,22 +59,23 @@ void Score::cmdJoinMeasure(Measure* m1, Measure* m2)
     if (m2->isMMRest()) {
         m2 = m2->mmRestLast();
     }
-    startCmd();
 
-    deselectAll();
+    mScore->startCmd();
```

Comment:
It looks like we should not call `startCmd` and `endCmd` inside this method. They are already called outside it.

Reply:
@cbjeukendrup 
Removing them caused utest spanners fails.
So I added them into test file.
Not sure, if it is OK (in case, if user would call joinMeasure or splitMeasure from command line)